etna/records/models.py

```diff
@@ -63,9 +72,19 @@ def availability_condition_category(self) -> str:
     def get_gtm_content_group(self) -> str:
         """
         Overrides DataLayerMixin.get_gtm_content_group() to
-        return the name of the class.
+        return content group otherwise the name of the class.
         """
-        return self.__class__.__name__
+        if (
+            self.catalogue_source == "CAT"
+            and self.repo_summary_title == "The National Archives"
```

Comment:
@TNA-Allan is this second condition necessary? What are the other potential values for `catalogue_source == "CAT"`?

Reply:
Does not look necessary, but I am confirming.

Reply:
There are other potential values from CAT. However removed condition as I have confirmed that CAT will apply to all TNA. Also updated the spreadsheet Custom Dimension for catalogue pages accordingly.

Reply:
This change is done